### ISSUE TITLE
Recent bug makes some items invisible.

### ISSUE THREAD

**Issue by RUTHINAS** [slowtorta/turtlewow-bug-tracker#5022]
**Please, provide a clear description what the bug is:**
Starting today, a large portion of the gear my characters are wearing has become invisible. I don't know if it only applies to transmogrified gear – and even then it doesn't seem to apply to every transmogrified piece – but in my case all the invisible items are ones that have been transmogrified.

The items show up normally in the character select screen, but after logging in the character panel is showing empty slots (which do display the item tooltips once you mouse over them) and the item model is absent from the character.

I already tried to correct it by restarting the game with the WDB folder cleared, to no effect. The issue was present on my other characters as well. Changing continents or zoning into an instance has no effect.

After the durability of the items changed, the items reverted to their original, untransmogrified appearance. However, after re-equipping them they again disappeared, meaning that there is no way of making the outfits show up as they are meant to be.

![screenshot](https://github.com/slowtorta/turtlewow-bug-tracker/assets/94254826/f4de0d76-862a-45f7-b8c6-0172a4fb00c8)

**Expected behavior. Describe how it should work:**
All items should be visible.

**Comment by Jan125** [comment]
Of note:
-Any model change auras (druid shapeshift, illusions) applied after initial load cause your character to become INVISIBLE, without a name or any attachment points
--The only thing visible is your shadow
--This only applies when the model change aura actually changed your model (druid in skeleton costume stay in skeleton model when shapeshifting and stay visible)
--Invisibility is determined on the model change happening AFTER your character has been loaded by you or other people's clients
--Taking off transmogged gear does not fix the issue immediately, you have to log out after taking it off
--When logging in without transmogged gear, everything behaves as it should
--Putting on transmogged gear again will cause issues to arise again

**Comment by Zmitrok1984** [comment]
I also have the same problem: xmoged items are invisible on character
![image](https://github.com/slowtorta/turtlewow-bug-tracker/assets/99164611/4acb39a3-c0f7-4fcf-9e74-91867c1fce9e)
![image](https://github.com/slowtorta/turtlewow-bug-tracker/assets/99164611/cb5b1c7b-f99e-44a0-901e-f291c84e0355)
![image](https://github.com/slowtorta/turtlewow-bug-tracker/assets/99164611/3b500fbb-388a-4851-89e9-5a30623424a4)

**Comment by Zmitrok1984** [comment]
I did some testing and it is looks like, that problem was happen with items, that were xmoged long time ago.
I removed xmog from my invisible sword -> it became visible with his real looks.
I xmoged sword agian as it was -> xmog appied to it and sword is visible.
So, question is possible to send me some fashion coins to re-apply xmog? Can I create a ticket with describing that problem to GM?

**Comment by ashphoenixblack** [comment]
I'm also having this problem. I tested it out further, after reading this thread. I purchased a brand new item & tmog'ed it. After tmog, same problem all over again. My character is a druid & shape-shifting does also make him invisible & also his mount & flying mounts. He only becomes visible again by using hearthstone or leaving a battleground. Travelling by flight or boat doesn't help

**Comment by RUTHINAS** [comment]
> I also have a separate issue with a bug for the quest to get Fashion coins. It affects all my character & has been happening for months now. Any char, either faction, both servers. I abandoned the quest once & ever since, the "continue" for "Fashion Demands Sacrifices" is greyed out & cant be selected. I'm especially disappointed by this issue following me to the new server, where there is no turtle mode & began after the 5yr anniversary so it's basically impossible to get any coins without buying a bag from the donation shop. I just spent a lot there during the sale & can't justify spending more for a while.

This sounds like an issue in the user interface. I would try clearing the Interface and WTF folders and giving it another try.

**Comment by ashphoenixblack** [comment]
Thank you! I will definitely try that! 
    On Thursday, November 2, 2023 at 02:59:59 PM EDT, RUTHINAS ***@***.***> wrote:  
 
 



I also have a separate issue with a bug for the quest to get Fashion coins. It affects all my character & has been happening for months now. Any char, either faction, both servers. I abandoned the quest once & ever since, the "continue" for "Fashion Demands Sacrifices" is greyed out & cant be selected. I'm especially disappointed by this issue following me to the new server, where there is no turtle mode & began after the 5yr anniversary so it's basically impossible to get any coins without buying a bag from the donation shop. I just spent a lot there during the sale & can't justify spending more for a while.


This sounds like an issue in the user interface. I would try clearing the Interface and WTF folders and giving it another try.

**Comment by ashphoenixblack** [comment]
That seems to just break the whole game. Perhaps I'm doing it wrong? I've never tried messing with those folders before...
    On Thursday, November 2, 2023 at 02:59:59 PM EDT, RUTHINAS ***@***.***> wrote:  
 
 



I also have a separate issue with a bug for the quest to get Fashion coins. It affects all my character & has been happening for months now. Any char, either faction, both servers. I abandoned the quest once & ever since, the "continue" for "Fashion Demands Sacrifices" is greyed out & cant be selected. I'm especially disappointed by this issue following me to the new server, where there is no turtle mode & began after the 5yr anniversary so it's basically impossible to get any coins without buying a bag from the donation shop. I just spent a lot there during the sale & can't justify spending more for a while.


This sounds like an issue in the user interface. I would try clearing the Interface and WTF folders and giving it another try.
